spec-tests/src/test/java/org/candlepin/spec/CloudRegistrationSpecTest.java

```diff
@@ -454,6 +454,23 @@ public void shouldNotExportCertificatesWithAnonymousTokenForUnknownAnonCloudCons
             .exportCertificates(StringUtil.random("unknown"), null));
     }
 
+    @Test
+    public void shouldCreateConsumerForV2()
+        throws Exception {
+        ApiClient adminClient = ApiClients.admin();
+        HostedTestApi upstreamClient = adminClient.hosted();
+        CloudRegistrationClient cloudRegistration = adminClient.cloudAuthorization();
+        OwnerDTO owner = upstreamClient.createOwner(Owners.random());
+        String token = cloudRegistration.cloudAuthorize(generateToken(owner.getKey(),
```

Comment:
.cloudAuthorizeV2 instead of cloudAuthorize

Reply:
done